src/sentry/static/sentry/app/views/releasesV2/detail/overview/otherProjects.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import {Location} from 'history';
+
+import {t, tn} from 'app/locale';
+import space from 'app/styles/space';
+import Link from 'app/components/links/link';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
+import {ReleaseProject} from 'app/types';
+import {IconProject} from 'app/icons';
+import Button from 'app/components/button';
+
+import {SectionHeading, Wrapper} from './styles';
+
+type Props = {
+  projects: ReleaseProject[];
+  location: Location;
+};
+
+type State = {
+  collapsed: boolean;
+};
+
+class OtherProjects extends React.Component<Props, State> {
+  state = {
+    collapsed: true,
+  };
+
+  static MAX_WHEN_COLLAPSED = 5;
+
+  onCollapseToggle = () => {
+    this.setState(prevState => ({
+      collapsed: !prevState.collapsed,
+    }));
+  };
+
+  render() {
+    const {projects, location} = this.props;
+    const {collapsed} = this.state;
+    const canExpand = projects.length > OtherProjects.MAX_WHEN_COLLAPSED;
+    let projectsToRender = projects;
+
+    if (collapsed && canExpand) {
+      projectsToRender = projects.slice(0, OtherProjects.MAX_WHEN_COLLAPSED);
+    }
+
+    const numberOfCollapsedProjects = projects.length - projectsToRender.length;
+
+    return (
+      <Wrapper>
+        <SectionHeading>
+          {tn(
+            'Other Project for This Release',
+            'Other Projects for This Release',
+            projects.length
+          )}
+        </SectionHeading>
+        {projectsToRender.map(project => (
+          <Row key={project.id}>
+            <Link
+              to={{
+                pathname: location.pathname,
+                query: {...location.query, project: project.id, yAxis: undefined},
+              }}
+            >
+              <StyledIconProject size="sm" />
```

Comment:
@matejminar Would it be better to use `ProjectBadge`?

Reply:
That's the way it's designed in Figma (https://www.figma.com/file/HmboUgnZ8ywVeo56R1pbl6/Releases-V2?node-id=152%3A667)
@doralchan what do you think?